src/main/java/net/floodlightcontroller/core/internal/Controller.java

```diff
@@ -678,6 +682,9 @@ public void init(Map<String, String> configParams) throws FloodlightModuleExcept
         // Switch Service Startup
         switchService.registerLogicalOFMessageCategory(LogicalOFMessageCategory.MAIN);
         counters = new ControllerCounters(debugCounterService);
+
+//        pktinProcTimeService.bootstrap();
```

Comment:
Try and remove commented out code, unless it’s a TODO

Reply:
It is not a TODO, I forgot to remove it. Thanks!